packages/react-components/react-charts-preview/library/src/components/LineChart/LineChart.tsx

```diff
@@ -37,6 +37,7 @@ import {
   isRtl,
   formatDate,
 } from '../../utilities/index';
+import { ResponsiveContainer } from '../CommonComponents/ResponsiveContainer';
```

Comment:
move responsive container to cartesian chart

Reply:
done